theano/tensor/tests/test_extra_ops.py

```diff
@@ -33,6 +35,123 @@ def test_cpu_contiguous():
     assert f(a_val, 3).flags['C_CONTIGUOUS']
 
 
+class TestSearchsortedOp(utt.InferShapeTester):
+
+    def setUp(self):
+        super(TestSearchsortedOp, self).setUp()
+        self.op_class = SearchsortedOp
+        self.op = SearchsortedOp()
+
+        self.x = T.vector('x')
+        self.v = T.tensor3('v')
+
+        self.a = np.random.random(100).astype(config.floatX)
+        self.b = np.random.random((10, 20, 5)).astype(config.floatX)
+        self.idx_sorted = np.argsort(self.a)
+
+    def tearDown(self):
+        self.x = None
+        self.v = None
+        self.a = None
+        self.b = None
+        self.idx_sorted = None
+
+    def test_searchsortedOp_on_sorted_input(self):
+        f = theano.function([self.x, self.v], searchsorted(self.x, self.v),
+                            mode="DebugMode")
```

Comment:
don't put a mode here. In our daily buildbot, we test with many different modes.